bin/test

```diff
@@ -6,7 +6,7 @@ flake8 --ignore=E501
 # Skip tests/test_destalinator.py due to heavy mocking. See https://github.com/jendrikseipp/vulture/issues/95
 # Skip utils/slack_logging.py due to implementing Handler#emit but never calling directly
 # Skip scheduler.py due to unused lambda entrypoint handler
```

Comment:
Should you update the comments for this too? They mention skipping things that it seems like you're no longer skipping.